test/e2e/v1beta1/scripts/gh-actions/run-e2e-experiment.py

```diff
@@ -0,0 +1,293 @@
+    logging.info(
+        "Creating Experiment: {}/{} with MaxTrialCount: {}, ParallelTrialCount: {}".format(
+            exp_namespace,
+            exp_name,
+            experiment.spec.max_trial_count,
+            experiment.spec.parallel_trial_count,
+        )
+    )
+
+    # Wait until Experiment reaches Succeeded condition.
+    katib_client.create_experiment(experiment, exp_namespace)
+    experiment = katib_client.wait_for_experiment_condition(
+        exp_name, exp_namespace, timeout=EXPERIMENT_TIMEOUT
+    )
+
+    # Test resume feature for "FromVolume" and "LongRunning" Experiments.
+    # TODO (andreyvelich): Once we change the default resume policy to "Never",
+    # also test restart for "LongRunning" Experiment here instead of "random".
+    # Ref: https://github.com/kubeflow/katib/issues/2055
+    if exp_name == "from-volume-resume" or exp_name == "random":
+        max_trial_count = experiment.spec.max_trial_count + 1
+        parallel_trial_count = experiment.spec.parallel_trial_count + 1
+        logging.info(
+            f"Restarting Experiment {exp_namespace}/{exp_name} "
+            f"with MaxTrialCount: {max_trial_count} and ParallelTrialCount: {parallel_trial_count}"
+        )
+
+        # Modify Experiment budget.
+        katib_client.edit_experiment_budget(
+            exp_name, exp_namespace, max_trial_count, parallel_trial_count
+        )
+        # Wait until Experiment is Restarted.
+        katib_client.wait_for_experiment_condition(
+            exp_name,
+            exp_namespace,
+            constants.EXPERIMENT_CONDITION_RESTARTING,
+            EXPERIMENT_TIMEOUT,
+        )
+        # Wait until Experiment is Succeeded.
+        experiment = katib_client.wait_for_experiment_condition(
+            exp_name, exp_namespace, timeout=EXPERIMENT_TIMEOUT
+        )
+
+    # Verify the Experiment results.
+    verify_experiment_results(katib_client, experiment, exp_name, exp_namespace)
+
+    # Describe the Experiment and Suggestion.
+    logging.info(
+        os.popen(f"kubectl describe experiment {exp_name} -n {exp_namespace}").read()
+    )
+    logging.info("---------------------------------------------------------------")
+    logging.info("---------------------------------------------------------------")
+    logging.info(
+        os.popen(f"kubectl describe suggestion {exp_name} -n {exp_namespace}").read()
+    )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--experiment-path",
+        type=str,
+        required=True,
+        help="Path to the Katib Experiment.",
+    )
+    parser.add_argument(
+        "--verbose",
+        type=str,
+        default=True,
+        choices=("True", "False"),
+        required=False,
+        help="Verbose output for the Katib E2E test",
+    )
+    args = parser.parse_args()
+
+    if args.verbose == "False":
+        logging.getLogger().setLevel(logging.WARNING)
```

Comment:
Instead, set default to Debug and change to INFO when verbose is False?

Also, set start/end logs to INFO and rest all logs to DEBUG?